src/LundPlane.jl

```diff
@@ -0,0 +1,143 @@
+  - `h_pt`: harder branch pt
+  - `s_pt`: softer branch pt
+  - `z`: momentum fraction of the softer branch
+  - `delta`: angular distance between branches
+  - `kt`: transverse momentum of the softer branch relative to the harder
+  - `psi`: azimuthal angle between branches
+  - `kappa`: z * ΔR
+"""
+function generate_lund_projection(jet::PseudoJet, cs::ClusterSequence{PseudoJet})
+
+    # Recluster the input jet using Cambridge/Aachen with large R
+    reconstructed_cluster_seq = recluster(jet, cs; algorithm = JetAlgorithm.CA,
+                                          R = 1000.0)
+    reconstructed_jet = inclusive_jets(reconstructed_cluster_seq; T = PseudoJet)[1]
+
+    lundPoints = Vector{NamedTuple}()
+    p1 = reconstructed_jet
+
+    while true
+        # Attempt to decluster p1 into p1 and p2
+        p1, p2 = decluster(p1, reconstructed_cluster_seq)
+
+        if isnothing(p1) || isnothing(p2)
+            break  # No further parents
+        end
+
+        # Basic kinematic quantities
+        harder_pt = pt(p1)
+        softer_pt = pt(p2)
+
+        Δ = deltaR(p1, p2)                            # Angular distance
+        kt = pt(p2) * Δ                               # Relative transverse momentum
+        z = pt(p2) / (pt(p1) + pt(p2))                # Momentum sharing
+        psi = atan(rapidity(p2) - rapidity(p1), delta_phi(p2, p1))  # Azimuthal angle in (Δφ, Δy) plane
+        kappa = z * Δ                                 # Log-polar projection
+
+        # Store one step of the declustering
+        temp = (h_pt = harder_pt,
+                s_pt = softer_pt,
+                z = z,
+                delta = Δ,
+                kt = kt,
+                psi = psi,
+                kappa = kappa)
+
+        push!(lundPoints, temp)
+    end
+
+    return lundPoints
+end
+
+"""
+generate_average_lund_image(njets::Int, delta_array::Vector{Vector{Real}}, 
+                            kt_array::Vector{Vector{Real}}; xrange::Tuple{Real}, 
+                            yrange::Tuple{Real}, bins::Int) -> (xgrid, ygrid, avg_image)
+
+Computes an average Lund image from a set of jets by binning the (log(1/ΔR), log(kt))
+coordinates into a fixed-size 2D histogram. Each jet's histogram is normalized and
+then averaged across all jets.
+
+Arguments:
+- `njets`: Number of jets
+- `delta_array`: Vector of vectors, where each inner vector contains ΔR values for a jet
+- `kt_array`: Vector of vectors, where each inner vector contains kt values for a jet
+- `xrange`: Tuple defining the x-axis (log(1/ΔR)) range
+- `yrange`: Tuple defining the y-axis (log(kt)) range
+- `bins`: Number of bins along each axis
+
+Returns:
+- A tuple `(xgrid, ygrid, avg_image)` where `xgrid` and `ygrid` are coordinate axes labels and
+  `avg_image` is the averaged 2D histogram as a matrix
+"""
+function generate_average_lund_image(njets::Int, delta_array::Vector{Vector{Real}},
+                                     kt_array::Vector{Vector{Real}}; xrange = (0.0, 9.0),
```

Comment:
The default ranges here are very large -- even in your example, the interesting part of the 2D plot doesn't extend beyond ∆R~4. 

Something like ∆R~4.5 is already smaller than the inner detector pixel size at the LHC. I don't think it is useful to have the example go much more colinear than that.